shared/settings/feedback/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export const getOtherErrorInfo = (err: Error) => {
```

Comment:
what is this file for. we don't want to mask the index.desktop/index.native files

Reply:
oops this was meant to be in the `util.js` file but will merge the desktop/native index files